opal/mca/hwloc/hwloc201/Makefile.am

```diff
@@ -25,7 +25,9 @@ EXTRA_DIST = \
         hwloc/doc/README.txt \
         hwloc/contrib/systemd/README.txt \
         hwloc/tests/README.txt \
-        hwloc/utils/README.txt
+        hwloc/utils/README.txt \
+	hwloc/netloc/README.txt \
+	hwloc/contrib/misc/README.txt
```

Comment:
There's a change in indentation between the last 2 lines -- should probably make it the same as all the other lines in `EXTRA_DIST`.